presto-base-jdbc/src/main/java/io/prestosql/plugin/jdbc/BaseJdbcConfig.java

```diff
@@ -37,6 +39,8 @@
     private boolean cacheMissing;
 
     @NotNull
+    // Some drivers match case insensitive in Driver.acceptURL
+    @Pattern(regexp = "^jdbc:[a-z0-9]+:(?s:.*)$", flags = CASE_INSENSITIVE, message = "Invalid JDBC URL: must match the following regular expression {regexp}")
```

Comment:
The default message is good enough:

```suggestion
    @Pattern(regexp = "jdbc:[a-z0-9]+:(?s:.*)", flags = CASE_INSENSITIVE)
```